### PR TITLE
Addition of version argument to installation script

### PR DESCRIPTION
**Pull request by AhmedLotfy02**
## Related Issue
  - 

Closes: #1639

#### Describe the changes you've made
Addition of version argument to installation script

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Please let us know if any test cases are added

Please describe the tests(if any). Provide instructions how its affecting the coverage.

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
A clear and concise description of it.

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.

## Screenshots (if any)

|        Original         |          Updated           |
|:-----------------------:|:--------------------------:|
| **original screenshot** | <b>updated screenshot </b> |

### CODE REVIEW

**Review by slayerjain — CHANGES_REQUESTED** [review]
we should also add some docs about this usage in the docs repo.

File: keploy.sh [hunk 1]
```diff
@@ -2,20 +2,32 @@
...
+                ;;
         esac
     done
-
+    if [ "$VERSION" = "latest" ]; then
+        # If version is "latest", use the latest release
+        VERSION_TAG="latest"
+    else
+        # If specific version is provided, use that version
+        VERSION_TAG="v$VERSION"
```
[the remark this answers is not in the document]
Comment (AhmedLotfy02) [reply keploy.sh]: @slayerjain  Done

File: keploy.sh [hunk 2]
```diff
@@ -2,20 +2,32 @@
...
+    fi
     install_keploy_arm() {
-        curl --silent --location "https://github.com/keploy/keploy/releases/latest/download/keploy_linux_arm64.tar.gz" | tar xz -C /tmp
+        curl --silent --location "https://github.com/keploy/keploy/releases/download/$VERSION_TAG/keploy_linux_arm64.tar.gz" | tar xz -C /tmp
```
[the remark this answers is not in the document]
Comment (AhmedLotfy02) [reply keploy.sh]: @slayerjain Done adding it to amd section and docker section.

**Comment by AhmedLotfy02** [comment]
@slayerjain All edits requested are done.

**Comment by gouravkrosx** [comment]
@AhmedLotfy02 Please resolve the merge conflicts.

**Comment by AhmedLotfy02** [comment]
@gouravkrosx Done.

**Comment by shivamsouravjha** [comment]
@gouravkrosx any update on this?

**Comment by AhmedLotfy02** [comment]
@shivamsouravjha Done.

**Comment by shivamsouravjha** [comment]
closing the PR as issue is solved

**Pull request closed without merging**